lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -79,6 +79,10 @@ class CanvasKit {
   external int get LineThroughDecoration;
   // End of text decoration enum.
 
+  external SkTextDecorationStyleEnum get DecorationStyle;
+  external SkTextBaselineEnum get TextBaseline;
+  external SkPlaceholderAlignmentEnum get PlaceholderAlignment;
```

Comment:
Let's add tests for all of the new bindings in `canvaskit_api_test.dart`.

Reply:
Done.